### PR TITLE
Bah 8940 disabilities

### PR DESCRIPTION
**Pull request by aaronjlech**
## Description
Adds functionality to list all disabilities (new and rated). to be selected for cause of unemployability within the 8940.

## Testing done
Locally tested
Unit tests written

## Screenshots

**List of rated and new disabilities**

image

**Validation if none are selected**

image

**List of just rated disabilities(no new disabilities selected)**

image

**List of just new disabilities(no rated disabilities on user)**

image

## Acceptance criteria
- [x] Allow users to select new and/or rated disabilities as a related disability to their unemployability.
- [x] User is required to select at least one related disability. 

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/validations.js [hunk 1]
```diff
@@ -126,6 +126,29 @@ export const validateIfHasEvidence = (
   }
 };
 
+// Need the Lambda to pass the disability list type, so only 1 disability list has the error message.
+export const oneDisabilityRequired = disabilityList => (
```
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/validations.js]: does this need a unit test?

**Review by rianfowler — APPROVED** [review]
looks good. just one question and a couple of failures: 
```
AssertionError: expected [ Array(3) ] to be empty
at Context.<anonymous> (src/applications/disability-benefits/all-claims/tests/property-names.unit.spec.js:63:5)
```

```
/application/src/applications/disability-benefits/all-claims/tests/pages/unemployabilityDisabilities.spec.jsx

  21:18  error  mount() has no matching ReactWrapper.unmount()  va-enzyme/unmount

  37:18  error  mount() has no matching ReactWrapper.unmount()  va-enzyme/unmount

  58:18  error  mount() has no matching ReactWrapper.unmount()  va-enzyme/unmount

  74:18  error  mount() has no matching ReactWrapper.unmount()  va-enzyme/unmount
```

File: src/applications/disability-benefits/all-claims/validations.js (hunk 1 above)
Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/validations.js]: Unit tests added!

Comment (rianfowler) [reply src/applications/disability-benefits/all-claims/validations.js]: cool this looks ready to merge.

**Pull request merged**